docs/learn/learn-guides-accounts.md

```diff
@@ -18,14 +18,14 @@ description: Comprehensive guides for managing accounts, backups, and security u
 
 ## Account Address Format
 
-An account created on the relay chain can also be used on multiple chains in the ecosystem. More
+An account created on the Polkadot Hub can also be used on multiple chains in the ecosystem. More
```

Comment:
I wonder if its Polkadot Hub or Polkadot. I think Polkadot would make more sense for new users?

Reply:
I was wondering the same. We can just put Polkadot as it does eliminate confusion